packages/askui-nodejs/src/execution/client-interface.ts

```diff
@@ -2,9 +2,14 @@ import { CredentialArgs } from '../utils/http/credentials';
 import { AnnotationLevel } from './annotation-level';
 
 export interface ClientArgs {
+  // The adress of the askui UI Controller server.
```

Comment:
I think we should use javadoc comment style for all comments for official documentation as most generators that use these comments expect that as far as I know, e.g.,

```suggestion
  /**
    * The address of the askui UI Controller server.
    */
```